analyze_test.go

```diff
@@ -1067,3 +1067,183 @@ func TestAnalyze_nap_app_protect_reconnect_period_seconds(t *testing.T) {
 		})
 	}
 }
+
+//nolint:funlen
+func TestAnalyze_http3(t *testing.T) {
+	t.Parallel()
+	testcases := map[string]struct {
+		stmt    *Directive
+		ctx     blockCtx
+		wantErr bool
+	}{
+		"http3 ok": {
+			&Directive{
+				Directive: "http3",
+				Args:      []string{"on"},
+				Line:      5,
+			},
+			blockCtx{"http", "server"},
+			false,
+		},
+		"http3 not ok": {
+			&Directive{
+				Directive: "http3",
+				Args:      []string{"somevalue"},
+				Line:      5,
+			},
+			blockCtx{"http", "server"},
+			true,
+		},
+		"http3_hq ok": {
```

Comment:
What is this testing that the testcase starting on line 1079 is not covering? Looks identical to me

Reply:
oops! I forgot to change the directive name.   It should be `http3_hq`